src/components/button/buttonVariant.ts

```diff
@@ -0,0 +1,42 @@
+import { type VariantProps, cva } from 'class-variance-authority';
+
+export type BaseButtonProps = VariantProps<typeof button>;
+
+export const button = cva(['font-semibold', 'border'], {
+  variants: {
+    variant: {
+      contained:
+        'bg-button text-typography hover:bg-buttonHover focus:bg-buttonHover',
+      outlined: 'border-button text-button',
+      text: 'text-typography hover:bg-primary/50',
+      // **or**
+      // primary: "bg-blue-500 text-white border-transparent hover:bg-blue-600",
```

Comment:
can we have a reason  for this commented code

Reply:
no